routes/web.php

```diff
@@ -19,7 +19,18 @@
 
 Route::get('/home', 'HomeController@index')->name('home');
 
-//temporary
+//temporary dashoboard page rooting
 Route::get('/dashboard', function () {
     return view('dashboard');
 });
+
+//temporary admin_add_category page rooting
+Route::get('/admin_add_category', function () {
+    return view('admin.addCategory');
```

Comment:
Please don't create temporary routes, please use relevant routes, I suggest that you group admin routes, and better if you have proper routing naming.

For example:
```
Route::get('/admin/category/create', '...');
Route::post('/admin/category/store', '...');
```

this is more readable and organize to look at

Reply:
ah ok, if fixed in the later PR then it's ok